### PR TITLE
Added task id command and integration test

### PR DESCRIPTION
**Pull request by zhanggbj**
Hi dr.max @maximilien ,

The original PR was deleted by mistake. Please help to review this one. Thanks.

[#115478103](https://www.pivotaltracker.com/story/show/115478103)

### CODE REVIEW

**Comment by maximilien** [comment]
OK, so you have conflicts on this. I'll wait for you to resolve before reviewing. Thx.

**Comment by maximilien** [comment]
Please note here when you are ready for me to do another review. Thx.

File: cmds/bmp/task_command.go [hunk 1]
```diff
@@ -46,10 +49,35 @@ func (cmd taskCommand) Options() cmds.Options {
...
+	taskOutputResponse, err := cmd.bmpClient.TaskOutput(cmd.options.TaskID, level)
+	if err != nil {
+		return 1, err
+	}
+
+	if taskOutputResponse.Status != 200 {
+		return taskOutputResponse.Status, nil
+	}
+
+	fmt.Printf("Task output for ID %d with %s level", cmd.options.TaskID, level)
```
Comment (maximilien) [inline cmds/bmp/task_command.go:77]: Grace any output needs to be via cmd.Ui

**Comment by maximilien** [comment]
See my feedback. Please address. Also, this PR needs to be associated with a story number. If there is none then create a bug or chore. Thx.

File: cmds/bmp/task_command.go (hunk 1 above)
Comment (zhanggbj) [inline cmds/bmp/task_command.go:77]: Yes, I see, I will update with chore [#119336713](https://www.pivotaltracker.com/story/show/119336713)

**Comment by zhanggbj** [comment]
This PR is part of story [#115478103](https://www.pivotaltracker.com/story/show/115478103). 
This is ready for your review. Thanks:)

File: cmds/bmp/task_command.go (hunk 1 above)
Comment (maximilien) [inline cmds/bmp/task_command.go:77]: OK fine, but in the future, let's avoid having chores to fix things on fixes... This type of workflow encourages accumulation of technical debt. Generally a bad thing.

**Comment by maximilien** [comment]
OK cool. A minor comment which is a point of how we should plan our work in future. Cheers 🍻 

You also need a story ID for this PR.

**Pull request merged**